response/slack/client.py

```diff
@@ -72,7 +68,7 @@ def get_channel_id(self, name):
             try:
                 next_cursor = response["response_metadata"]["next_cursor"]
                 logger.info(f"get_channel_id - next_cursor == [{next_cursor}]")
-            except:
+            except LookupError:
```

Comment:
Just checking - was catching this specific exception intentional?

Don't think it matters, the error handling here isn't great anyway to be honest

Reply:
Yes. "Blind excepts" are banned under the linter to force more careful exception handling. In this case my assumption was that the error was missing details in the `response` dict, so I went with a `LookupError` as a fairly generic thing to catch.